### ISSUE TITLE
Registration Backend Cleanup - Post Reg2

### ISSUE THREAD

**Issue by Sepehr-Sobhani** [bcgov/cas-registration#2137]
#### Describe the task

Once we have fully implemented Reg part 2, we need to clean up the backend (API endpoints, services, and tests) and remove the codes not being used in Reg 2 and are specific to Reg 1.

#### Acceptance Criteria

- [x] No more V1 and V2 directories in the backend
- [x] Remove unused schema and services
- [x] Remove unused endpoints
- [x] Remove redundant tests

#### Additional context

- Add any other context about the task here.
- Or here

**Comment by Sepehr-Sobhani** [comment]
> Should this and [#2229](https://github.com/bcgov/cas-registration/issues/2229) go together?

No, but it's best to avoid working on both in parallel to prevent conflicts.

**Comment by Sepehr-Sobhani** [comment]
This is just removing Reg1 backend code.

**Comment by fviduya** [comment]
This ticket involves removing redundant reg1 code, did regression test to verify major workflows still work, [this is the document for reference](https://buttoninc-my.sharepoint.com/:w:/g/personal/francesca_button_is/EZEL17mV4eVJqCEtQN7N4pwBAARsOaZxQUUNUnlTDZkocg?e=jS1SmM):
- [x] Registering an operation
- [x] Selecting an operator
- [x] Adding an operator
- [x] Approving, declining, and editing users and access requests
- [x] Saving operation representative
- [x] Transferring operations and facilities
- [x] Adding/Editing a contact 

Moving to Pending PO. Thank you!
